lib/modules/dosomething/dosomething_reportback/dosomething_reportback.cron.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Provides cron job/drush command for dosomething_reportback.module.
+ */
+
+/**
+ * Implements hook_cron().
+ */
+function dosomething_reportback_cron() {
+  // Get all reportbacks updated in the past hour.
+  $results = db_query("SELECT nid, sum(quantity) as quantity
+                     FROM dosomething_reportback
+                     WHERE from_unixtime(updated) >= date_add(now(), INTERVAL '-1' hour)
```

Comment:
I thought it'd be better for performance: DB servers usually are more busy than webheads running cron jobs. But yes, your have legit point about time being in sync.